chsdi/locale/en/LC_MESSAGES/chsdi.po

```diff
@@ -1086,19 +1086,19 @@ msgid "ch.bafu.bundesinventare-amphibien_anhang4"
 msgstr "Amphibians appendix 4"
 
 msgid "ch.bafu.bundesinventare-amphibien_anhoerung"
-msgstr "Amphibien - Anhörung 2015"
```

Comment:
Yeah we lost it for some reason, same in my geojson PR. Simply put it back in the translation table.